packages/platform-sdk-profiles/src/profiles/profile.ts

```diff
@@ -190,6 +190,7 @@ export class Profile implements ProfileContract {
 		this.settings().set(ProfileSetting.TimeFormat, "h:mm A");
 		this.settings().set(ProfileSetting.UseCustomPeer, false);
 		this.settings().set(ProfileSetting.UseMultiPeerBroadcast, false);
+		this.settings().set(ProfileSetting.UseTestNetworks, true);
```

Comment:
```suggestion
		this.settings().set(ProfileSetting.UseTestNetworks, false);
```

This should be `false` by default so that the average user won't be presented with test networks and keeps wondering what they are. Test networks are explicitly for people that know what they should do with it.

Reply:
Thanks @faustbrian :+1: 

Fixing the tests and will mark it as ready to be merged